Userland/Games/2048/Game.h

```diff
@@ -44,7 +45,10 @@ class Game final {
         void add_tile(size_t row, size_t column, u32 value)
         {
             tiles[row][column] = value;
+            last_added_position = Tuple { row, column };
         }
+
+        Tuple<size_t, size_t> last_added_position { 0, 0 };
```

Comment:
Could we please not use `Tuple` in normal code?
it tends to make life/understanding very difficult in normal cases

Reply:
I originally added `Tuple` purely for metaprogramming tasks, and I think it should stay that way

Reply:
I will say that I originally used Tuple because I expected it to already have `operator==()` implemented and for no other reason. But yes, fair point.

Reply:
Thanks for keeping the code readable lol